project_restorer/src/main/scala/PlutoCoreMessageProcessor.scala

```diff
@@ -23,6 +23,8 @@ class PlutoCoreMessageProcessor(mxsConfig:MatrixStoreConfig)(implicit mat:Materi
                                                              ec:ExecutionContext) extends MessageProcessor {
   private val logger = LoggerFactory.getLogger(getClass)
 
+  private val statusesMediaNotRequired = List("held", "completed", "killed") // Could we use com.gu.multimedia.storagetier.plutocore.EntryStatus as source here. Are those the canonical states?
```

Comment:
Yes they are canonical, I'd recommend using those instead to avoid any potential typo or capitalization issues when editing the code

Reply:
Alrighty!